### PR TITLE
Switching active conversation using cmd+[1-9].

### PR DESCRIPTION
**Pull request by cvx**
It depends on a css selector ("._1ht1") which might change at any time, breaking this functionality.
Any ideas on what's the best way to implement such interactions between this app and the messenger?

Also, I'm not sure if this: `((AppDelegate *) self.delegate)` is how one should do things (\* insert no-idea-what-i'm-doing-dog-jpg here *).

### CODE REVIEW

File: Messenger/AppDelegate.mm [hunk 1]
```diff
@@ -105,6 +105,10 @@ - (void)applicationDidFinishLaunching:(NSNotification *)aNotification {
   _lastNotificationCount = @"";
 }
 
+- (void)setActiveConversationAtIndex:(NSString *)index {
+  [_webView.windowScriptObject evaluateWebScript:
+   [NSString stringWithFormat:@"document.querySelector('._1ht1:nth-of-type(%@) > a').click();", index]];
```
Comment (rsms) [inline Messenger/AppDelegate.mm]: I'm afraid this won't work. These CSS classes are automatically generated and might change at any point in time. Unless there are human-made identifiers, we won't be able to reliably find these DOM elements.

File: Messenger/FBMApplication.m [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#import "FBMApplication.h"
+#import "AppDelegate.h"
+
+@implementation FBMApplication
+
+@dynamic delegate;
```
Comment (rsms) [inline Messenger/FBMApplication.m]: Why is this needed?

File: Messenger/AppDelegate.mm (hunk 1 above)
Comment (cvx) [inline Messenger/AppDelegate.mm]: We could change that to:

```
li:nth-child(%@) > [data-reactid]:first-child
```

which is a bit less likely to break

File: Messenger/FBMApplication.m (hunk 2 above)
Comment (cvx) [inline Messenger/FBMApplication.m]: It is not needed at all. :) Just some leftovers from my code experiments. I'll amend this commit.

File: Messenger/AppDelegate.mm (hunk 1 above)
Comment (rsms) [inline Messenger/AppDelegate.mm]: SGTM. Are there no other LI elements?

Comment (cvx) [inline Messenger/AppDelegate.mm]: There are. We're depending here on the order of elements in the DOM tree (taking the first matching element). But unless there are major changes to the Messenger layout, we should be fine.
